.github/workflows/ecosystem-tools.yaml

```diff
@@ -51,3 +51,6 @@ jobs:
       - name: Run anonymous-push-pull tests
         run: |
             make anonymous-push-pull
```

Comment:
we should really rename these Makefile targets with a test-* prefix
make test-annotations test-anon-push-pull etc
@nicoldr can we fix this also, your PR otherwise looks ok

Reply:
Added "test-" before every target of that kind